.idea/.gitignore

```diff
@@ -1,28 +1,3 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
-
-# Above is default, IntelliJ-generated config. Below is our custom config.
-# Inspired by:
-# - https://github.com/Vadorequest/JetBrains-Intellij-IDEA-.gitignore-best-practices
-# - https://github.com/salarmehr/idea-gitignore
-# - https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
-
-dataSources/
-dataSources.local.xml
-misc.xml
-workspace.xml
-
-google-java-format.xml
-inspectionProfiles/
-
-compiler.xml
-deploymentTargetSelector.xml
-gradle.xml
-kotlinc.xml
-vcs.xml
```

Comment:
I'd prefer to leave it as it is. Feel free to add new files to this gitignore though.